releng/platform-definitions/platform-definition-2021-03/platform-definition-2021-03.target

```diff
@@ -46,6 +46,9 @@
             <unit id="org.adoptopenjdk.jemmy-browser" version="2.0.0"/>
             <unit id="org.adoptopenjdk.jemmy-core" version="2.0.0"/>
             <unit id="org.adoptopenjdk.jemmy-swt" version="2.0.0"/>
+            <unit id="javax.servlet-api" version="3.1.0"/>
+            <unit id="javax.websocket-api" version="1.1.0"/>
+            <unit id="org.eclipse.jetty.websocket.server" version="9.4.43.v20210629"/>
```

Comment:
It seems there's still references to jetty 9 in older platform definition.

I'm not familiar with eclipse based applications, but should those target be removed once the platform is upgraded to a more recent version ?

Reply:
🙈  Cleaned up now, I was just fiddling around with this config trying to get the dependencies to resolve properly and I was just updating the active one.